gordian/gordian.py

```diff
@@ -155,7 +174,8 @@ def transform(args, transformations, repositories, pr_created_callback):
             repo.bump_version(args.dry_run)
             if not args.dry_run:
                 try:
-                    pull_request = repo.create_pr(args.pr_message, '', args.target_branch, args.pr_labels)
+                    pull_request = repo.create_pr(args.pr_message, pr_description, args.target_branch, args.pr_labels)
+                    pull_request.set_labels(*args.pr_labels)
```

Comment:
I think this was duplicated when rebasing possibly? The label is already being set here https://github.com/argoproj-labs/gordian/blob/master/gordian/repo.py#L204.